src/pages/settings/Profile/Contacts/ContactMethodDetailsPage.js

```diff
@@ -230,6 +230,7 @@ class ContactMethodDetailsPage extends Component {
         const isDefaultContactMethod = this.props.session.email === loginData.partnerUserID;
         const hasMagicCodeBeenSent = lodashGet(this.props.loginList, [contactMethod, 'validateCodeSent'], false);
         const isFailedAddContactMethod = Boolean(lodashGet(loginData, 'errorFields.addedLogin'));
+        const isFailedRemovedContactMethod = !!lodashGet(loginData, 'errorFields.deletedLogin');
```

Comment:
The variable (`isFailedAddContactMethod`) above uses `Boolean`. Let's be consistent, use `Boolean` or `!!` for both variables. Also, I would suggest to add a default value.
```suggestion
        const isFailedRemovedContactMethod = Boolean(lodashGet(loginData, 'errorFields.deletedLogin'));
```

Reply:
you are right, I automatically used my preferred way of bool type coercion :)